### PR TITLE
SL-1951 Update indexes on node_version_history_changelog table

### PR DESCRIPTION
**Pull request by pytlesk4**
[SL-1951](https://stoplightio.atlassian.net/browse/SL-1951)

### CODE REVIEW

File: db/migrate/20190301202346_update_node_version_history_changelog_indexes.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+class UpdateNodeVersionHistoryChangelogIndexes < ActiveRecord::Migration
+  include Gitlab::Database::MigrationHelpers
+
+  DOWNTIME = false
+
+  def up
+    if Gitlab::Database.postgresql?
+      execute <<-SQL
+        CREATE UNIQUE INDEX IF NOT EXISTS node_version_history_changelog_node_version_hisotry_id_message_idx ON node_version_history_changelog USING btree (node_version_history_id, message);
```
Comment (chohmann) [inline db/migrate/20190301202346_update_node_version_history_changelog_indexes.rb]: super minor, but index name has a typo. 

node_version_history_changelog_node_version_**hisotry**_id_message_idx

**Review by chohmann — DISMISSED** [review]
One minor typo.

File: db/migrate/20190301202346_update_node_version_history_changelog_indexes.rb (hunk 1 above)
Comment (pytlesk4) [reply db/migrate/20190301202346_update_node_version_history_changelog_indexes.rb]: Good catch, fixed!

**Review by chohmann — APPROVED** [review]
🚢 it

**Pull request merged**